lms/static/js/dashboard/legacy.js

```diff
@@ -103,12 +103,12 @@
                  diagAttr['data-track-info'] = gettext('Are you sure you want to unenroll from %(courseName)s ' +
                                                    '(%(courseNumber)s)?');
              } else if (showRefundOption) {
-                 diagAttr['data-track-info'] = gettext('Are you sure you want to unenroll from the verified ' +
-                                                   '%(certNameLong)s  track of %(courseName)s  (%(courseNumber)s)?');
+                 diagAttr['data-track-info'] = gettext('Are you sure you want to unenroll from the ' +
+                                                   'Statement of Accomplishment track of %(courseName)s  (%(courseNumber)s)?');
                  diagAttr['data-refund-info'] = gettext('You will be refunded the amount you paid.');
              } else {
-                 diagAttr['data-track-info'] = gettext('Are you sure you want to unenroll from the verified ' +
-                                                   '%(certNameLong)s track of %(courseName)s (%(courseNumber)s)?');
+                 diagAttr['data-track-info'] = gettext('Are you sure you want to unenroll from the ' +
+                                                   'Statement of Accomplishment track of %(courseName)s (%(courseNumber)s)?');
```

Comment:
There is a config `CERT_NAME_LONG` in common.py. Please check if we can achieve this change using that config instead of hardcoding it here.

Reply:
Yeah working on it. but we still have to make some changes in the js file as that file still has the word "verified" which needs to be updated in theme.